lua/tardis/settings/sh_global_settings.lua

```diff
@@ -44,6 +44,93 @@ TARDIS:AddSetting({
     name="MaxHealth",
 })
 
+TARDIS:AddSetting({
+    id="long_repair",
+    type="bool",
+    value=true,
+
+    class="global",
+    convar = {
+        name = "tardis2_long_repair",
+        flags = CONVAR_FLAGS,
+        desc = "TARDIS - Whether repairing process lasts 60 seconds",
+    },
+
+    option = true,
+    section=SETTING_SECTION,
+    name="WaitLongForRepair",
+})
+
+TARDIS:AddSetting({
+    id="artron_energy",
+    type="bool",
+    value=true,
+
+    class="global",
+    convar = {
+        name = "tardis2_artron_energy",
+        flags = CONVAR_FLAGS,
+        desc = "TARDIS - Whether artron energy is enabled",
+    },
+
+    option = true,
+    section=SETTING_SECTION,
+    subsection="ArtronEnergy",
+    name="ArtronEnergyEnabled",
+})
+
+TARDIS:AddSetting({
+    id="artron_energy_start",
+    type="list",
+    value=0.25,
+    get_values_func = function()
+        return {
+            { "0%", 0.0 },
+            { "25%", 0.25 },
+            { "50%", 0.5 },
+            { "75%", 0.75 },
+            { "100%", 1.0 },
+        }
+    end,
+
+    class="global",
+    convar = {
+        name = "tardis2_artron_energy_start",
+        flags = CONVAR_FLAGS,
+        desc = "TARDIS - Initial amount of artron energy",
+    },
+
+    option=true,
+    section=SETTING_SECTION,
+    subsection="ArtronEnergy",
+    name="ArtronEnergyStart",
+})
+
+TARDIS:AddSetting({
```

Comment:
Should we instead consider making this a slider for drain rate multiplier instead with the default at 1? I think that might be easier for users to understand than a random number

Reply:
I've scaled down the numbers
The default value is now 100, the min. and max. values are 25 and 400
I didn't want to add %, since it would create confusion with "start value" persentages
But it should be more intuitive

Reply:
I think this works for now